resources/qml/Sidebar.qml

```diff
@@ -214,7 +214,7 @@ Rectangle
                 anchors.left: parent.left
                 anchors.leftMargin: model.index * (settingsModeSelection.width / 2)
                 anchors.verticalCenter: parent.verticalCenter
-                width: parent.width / 2
+                width: 0.5 * parent.width - (modesListModel.get(index).showFilterButton ? toggleFilterButton.width : 0)
```

Comment:
Ok, so what do you suggest?
```width: 0.5 * parent.width - toggleFilterButton.width``` ?
That will decrease the width of the "simple" button too. The idea is that the "simple" button is not changed, and the "advanced" button is slightly less wide to make room for the toggle button, but only when it is selected.